server/util.c

```diff
@@ -2190,6 +2190,219 @@ AP_DECLARE(char *) ap_escape_html2(apr_pool_t *p, const char *s, int toasc)
     x[j] = '\0';
     return x;
 }
+
+/* TODO: not sure about copyright&license of below code */
```

Comment:
This code is provided under the MIT license IIUC, so a note in the /LICENSE file should be fine I think. We can probably do that when merging.